Frontend/src/CaseCard.js

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Card from '@mui/material/Card';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+
+class CaseCard extends React.Component{
+
+    constructor(props) {
+        super(props);
+      }
```

Comment:
This constructor isn't required.

Reply:
Had it for if you implemented something that i might need it for, but seems like that's not the case so I'll remove it